src/ledger/assets/tokens.spec.ts

```diff
@@ -31,4 +31,12 @@ describe("Tokens", () => {
 			t1.sub(t2);
 		}, "subbing incompatible tokens should throw").throws;
 	});
+
+	it("compares tokens", () => {
+		expect(new Tokens([124n]).cmp(new Tokens([]))).to.eql("gt");
+		expect(new Tokens([124n]).cmp(new Tokens([124n]))).to.eql("eq");
+		expect(new Tokens([125n]).cmp(new Tokens([124n]))).to.eql("uncomparable");
+		expect(new Tokens([125n]).cmp(new Tokens([124n, 125n]))).to.eql("lt");
+		expect(new Tokens([126n]).cmp(new Tokens([124n, 125n]))).to.eql("uncomparable");
```

Comment:
```suggestion
		expect(new Tokens([126n]).cmp(new Tokens([124n, 125n]))).to.eql("uncomparable");
				expect(
			new Tokens([123n, 125n]).cmp(new Tokens([123n, 124n, 125n, 126n])),
		).to.eql("lt");
		expect(
			new Tokens([123n, 124n]).cmp(new Tokens([122n, 123n, 124n, 125n])),
		).to.eql("lt");
		expect(
			new Tokens([123n, 124n, 125n]).cmp(new Tokens([123n, 124n])),
		).to.eql("gt");
		expect(
			new Tokens([122n, 123n, 124n, 125n]).cmp(new Tokens([123n, 124n])),
		).to.eql("gt");
```

Just more edge cases to be on the safe side, because  I saw these ones failing when double checking our go-impl...

Reply:
And you apparently do not run prettier? We have a `.prettierrc` but it kicked in again.